docs/examples/composer_app/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "monolog/monolog": "1.3",
+        "marcelog/pami": "dev-master"
```

Comment:
Let's leave it like this, since what you say will really depend on the project that is including pami, and this is just a very basic example